packages/nx/src/project-graph/plugins/public-api.ts

```diff
@@ -182,3 +182,5 @@ export type NxPluginV2<TOptions = unknown> = {
  * A plugin for Nx
  */
 export type NxPlugin = NxPluginV2;
+
+export * from './plugins';
```

Comment:
Would we consider `getPlugins` as public-api? I think its more internal api right? We don't expect it to be used outside of `nx`